### DISCUSSION TITLE
Code access for Federated Training (task 1) and evaluation metrics?

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#12] [Q&A]
Is there any code implementation available for optimisation for task 1 which requires improving the weight aggregation and Is there any code implementation available for evaluation metrics namely Dice Similarity Coefficient (DSC), 95% Hausdorff distance (HD)?

**Comment by sarthakpati** [comment]
Hi @dskhanirfan,

Thank you for your interest in the FeTS Challenge!

Regarding your question about evaluation metrics' implementation, we are using the same code as [the BraTS challenge](https://cbica.github.io/CaPTk/BraTS_Metrics.html). If you need something quick and comparable (but **not equivalent**), you can check [medpy](https://pypi.org/project/MedPy/).

The first question regarding optimization is unfortunately not clear, could you please clarify or restate it? Most of the details you would need are in the [challenge notebook](https://github.com/FETS-AI/Challenge/blob/main/Task_1/FeTS_Challenge.ipynb).

Cheers,
Sarthak on behalf of the FeTS Challenge Organizing Committee
